### ISSUE TITLE
Improvements on the Development guides

### ISSUE THREAD

**Issue by xredo** [MarsBased/handbook#43]
Some first suggestions from a first (yet uncomplete) read. The job in the guides has been amazing so far 💪 Congrats to all the team!

## Organization

I would create a separate section for development, instead of having it inside the `Sections` in the handbook home page. The development guide is meant to be read frequently.

In this new section we can add both the main link to the development guide and the list of the specific guides for each technology. We can then clean the other sections. We can have the following structure.

- Sections
- Development guides
- Other guides
- Other useful resources

## Development: Async is the way

https://github.com/MarsBased/handbook/blob/1379e808ae292e8758ee471df19c02f6198174e0/sections/development.md#36-async-is-the-way

Next to the Basecamp notifications I would add:

> - Disable notifications on shared Slack channels (e.g: #random, #status, #marsbased). Or disable notifications on all channels if you are in many. If someone needs to let you notice something they will use mentions.

## Development: Speak like a martian

> Let everybody know your daily schedule so the team can know when it's best to reach to you. Update your project management tool frequently with your status to keep everybody in the loop. Use Slack instead if you need a more direct status report.

I would add more information and separate this into 2 blocks. When reading it for the first time I didn't understand well the status thing. I would rephrase it as:

> - Let everybody know your daily schedule so the team can know when it's best to reach you. Add this information in your Slack profile. Use the #status Slack channel and / or Slack built-in status messages to inform when you are not available.
> - Update the status of the task you are working on frequently. Use comments in the the project management tool to keep everybody in the loop. Use Slack instead if you need a more direct status report.

## React guidelines

https://github.com/MarsBased/handbook/blob/1379e808ae292e8758ee471df19c02f6198174e0/guides/development/react-guidelines.md

> Until now, we've used create-react-app to bootstrap React applications and we follow it's conventions.

This sentence makes me think as if now we are going to start doing things different. I would rephrase it to:

> We use create-react-app to bootstrap React applications and we follow it's conventions.

It's more direct, simpler and can't lead to confusion. What do you think?

## Testing guidelines

https://github.com/MarsBased/handbook/blob/1379e808ae292e8758ee471df19c02f6198174e0/guides/development/testing-guidelines.md

We should remove the first line as I imagine it was meant to guide the person writing the guide.

**Comment by danigb** [comment]
> This sentence makes me think as if now we are going to start doing things different.

I think we should start using next.js (or any of its derivatives) 😇  create-react-app worked well so far, but there are some things that next.js do better, specially regarding SSR and SEO

**Comment by xredo** [comment]
@danigb 

> I think we should start using next.js (or any of its derivatives) 😇 create-react-app worked well so far, but there are some things that next.js do better, specially regarding SSR and SEO

Great. Thanks for clarifying. This should be our next discussion to have it well defined once we start our next React project.

**Comment by oriolbcn** [comment]
> I would create a separate section for development, instead of having it inside the Sections in the handbook home page. The development guide is meant to be read frequently.

There needs to be an entry point from the main Handbook page, otherwise people will not be able to find it. The idea is to have a single link "Development" to direct to the Development section. In there, the Development section acts as an entry point of development-related info and guides.

> In this new section we can add both the main link to the development guide and the list of the specific guides for each technology. We can then clean the other sections. We can have the following structure.

Sounds interesting, I'll make an attempt.

Rest of comments look good! I have opened a PR with the changes, take a look when you can.
